Storage/SQL/SQLiteHistory.swift

```diff
@@ -941,29 +941,37 @@ extension SQLiteHistory: SyncableHistory {
 
         // We'll need to runQueryUnsafe so we get a LiveCursor, i.e. we don't get the cursor
         // contents into memory all at once.
-        return db.runQueryUnsafe(sql, args: nil, factory: rowIdentity) >>== { cursor in
-            defer { cursor.close() }
-
+        return db.runQueryUnsafe(sql, args: nil, factory: rowIdentity) { (cursor: Cursor<SDRow>) ->  [Int: [Visit]] in
```

Comment:
Nit: you have two spaces after the `->`.

Reply:
:+1: